spec/controllers/api/schedules/schedules_index_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'spec_helper'
+
+describe Api::SchedulesController do
+
+  include Devise::TestHelpers
+
+  describe '#index' do
+
+    let(:user) { FactoryGirl.create(:user) }
+
+    it 'lists all schedules for a user' do
+      sign_in user
+      schedules = FactoryGirl
+                    .create_list(:schedule, 2, user: user)
+                    .map(&:_id)
+                    .map(&:to_s)
+                    .sort
```

Comment:
This is an opinion that I don't agree with hound on. Need to eventually change our hound.yml